kafka_consumer/CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 <!-- towncrier release notes start -->
 
+## 4.4.0 / 2024-06-11
+
+***Added***:
+
+* Update confluent_kafka and librdkafka ([#17726](https://github.com/DataDog/integrations-core/pull/17726))
```

Comment:
```suggestion
* Update `confluent_kafka` and `librdkafka` ([#17726](https://github.com/DataDog/integrations-core/pull/17726))
```

This was addressed [here](https://github.com/DataDog/integrations-core/pull/17797/files#diff-c5def5f8172d6f11c1e96227d9a71f332edd56554c9214a1adcf63d1ece95678R9)